page-home.php

```diff
@@ -8,19 +8,19 @@
 
 <div class="background-image" style="background-image: url(<?php echo $src; ?>);"></div>
 <div class="container">
-    <div class="bgFeatureBox">
-        <a id="bgFeature" href="<?php the_permalink(); ?>"><?php the_title(); ?></a>
+    <div class="bgfeaturebox">
```

Comment:
The idea was that all 3 of these are the same thing. They share almost all styles and the only way in which they differ is that "their text alignment depends on their ordinal". So it's copacetic to share 1 class and make that single style depend on order (eg: `:nth-child`).